### ISSUE TITLE
No translated states in HA

### ISSUE THREAD

**Issue by TheFes** [home-assistant/intents#1228]
If I ask for the state of a light, I get translated states in an actual live HA environment.
However, for cover entities I get `closed` instead of `dicht`

Entity on Dashoard (which shows translated state)
![image](https://user-images.githubusercontent.com/28803438/224720827-d23030c3-54db-4aeb-829a-1513947b657b.png)

Result from Assist
![image](https://user-images.githubusercontent.com/28803438/224720931-a6cda008-e88e-45df-9094-13851adc13c5.png)

**Comment by jose1711** [comment]
Yeah, it would be great if it's possible to use `state_translated` inside the response. AFAIK the only workaround is something like this: https://github.com/home-assistant/intents/blob/e1354ad9381db24235b150951853de79f00ebb83/responses/pl/HassGetState.yaml#L9-L18

**Comment by jose1711** [comment]
Related: https://github.com/home-assistant/intents/issues/2899
